packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -2,254 +2,259 @@
   <div class="signup">
     <div class="signup--content">
       <div class="signup--form">
-        <form id="signupForm" @submit="submitForm">
-          <div class="row">
-            <label class="field-wrapper" for="email">
-              Work Email
-              <input
-                type="text"
-                id="email"
-                maxlength="128"
-                v-model="form.email.value"
-                :class="{ error: !form.email.isValid }"
-                @blur="validationService.checkEmailInput('email')"
-              />
-              <ul
-                class="error-color error-msg"
-                v-if="form.email.errorList.length"
-              >
-                <li
-                  class="error-color"
-                  v-for="(error, index) in form.email.errorList"
-                  v-bind:key="index"
-                >
-                  {{ error }}
-                </li>
-              </ul>
-            </label>
-          </div>
-          <div class="user-name">
+        <div v-if="isRegionLocked">
+          <p>We believe that you are located in a region recently impacted by US sanctions relating to Russia's invasion of Ukraine and are not able to process this request on that basis. This change in our policy is effective as of March 21, 2022. If you have any questions, contact support.</p>
```

Comment:
Should there be an email/link here for `contact support`?

Reply:
Copy was tweaking, legal has asked us to not link to support.